adam/perception/high_level_semantics_situation_to_developmental_primitive_perception.py

```diff
@@ -798,6 +799,8 @@ def _perceive_object(self, situation_object: SituationObject) -> None:
     def _instantiate_object_schema(
         self,
         schema: ObjectStructuralSchema,
+        # This will track what types of objects have which RgbColorPerceptions
+        sub_object_nodes_to_rgb: Dict[OntologyNode, RgbColorPerception],
```

Comment:
Prefer more general types like `Mapping` as function arguments (it gives the caller more flexibility)